tests/integration/020_api_key/main.tf

```diff
@@ -11,10 +11,20 @@ resource "env0_api_key" "test_api_key" {
 }
 
 resource "env0_api_key" "test_user_api_key" {
-  name = "my-little-user-api-key-${random_string.random.result}"
+  name              = "my-little-user-api-key-${random_string.random.result}"
   organization_role = "User"
 }
 
+resource "env0_team" "team_resource" {
+  name        = "team-with-api-key"
+  description = "description"
+}
+
+resource "env0_user_team_assignment" "api_key_team_assignment" {
```

Comment:
This is for testing team api-key assignment (Which is just user team assignment).

Reply:
not sure if it's part of the scope of this PR, but as part of #405 can we document somewhere that the `user_id` can also be an `api_key.id` ?